circuits/cpp/src/aztec3/circuits/kernel/private/common.cpp

```diff
@@ -61,15 +59,14 @@ void common_validate_call_stack(DummyBuilder& builder, PrivateCallData<NT> const
  * - https://discourse.aztec.network/t/spending-notes-which-havent-yet-been-inserted/180
  *
  * @param builder
- * @param storage_contract_address Contract address to use when siloing read requests
  * @param historic_private_data_tree_root This is a reference to the historic root which all
  * read requests are checked against here.
- * @param read_requests the commitments being read by this private call
+ * @param read_requests the commitments being read by this private call - transient reads here are
```

Comment:
Following our naming chat yesterday, I'll add pedantic naming comments if I see any. Feel free to rename in a separate PR, though. Whatever's easiest for you. 
```suggestion
 * @param read_requests the commitments being read by this private call - 'pending note reads' here are
```